README.md

```diff
@@ -106,6 +106,30 @@ Currently, we support the following types of assets:
 - [ERC-721](https://ethereum.org/en/developers/docs/standards/tokens/erc-721/) (Non-fungible tokens)
 - [ERC-1155](https://ethereum.org/en/developers/docs/standards/tokens/erc-1155/) (Multi-tokens)
 
+### Input encodings for deposits
+
+As explained above, when a portal teleports assets to the execution layer, it adds an input to the DApp's input box, which will then need to be interpreted and validated by the off-chain machine. In order to do that, the machine will need to understand and decode the input's payload.
+
+The input payloads for deposits are always specified as packed ABI-encoded parameters, as detailed below:
+
+| Asset | Payload |
+| :- | :- |
+| Ether | `address sender, uint256 value, bytes execLayerData` |
+| ERC-20 | `bool success, address token, address sender, uint256 amount, bytes execLayerData` |
+| ERC-721 | `address token, address sender, uint256 tokenId, bytes data` `*` |
+| ERC-1155 (single) | `address token, address sender, uint256 tokenId, uint256 value, bytes data` `*`|
+| ERC-1155 (batch) | `address token, address sender, bytes data` `**` |
```

Comment:
I feel like we need to explain each component of each input, but this table would turn too wide. Maybe we could try exploding it into several subsections. This will give more space for more detailed descriptions.

Reply:
I see what you mean, but I think it's more or less understandable as such. As you said, the table is already too big, and I also think that adding more subsections will be a little too much for a README, it's already really long. In any case, the meaning/description of each component is actually already included in the website API documentation, since they all derive from the arguments used when calling the deposit methods.

Reply:
Ok. I'm fine with this README being just a superficial documentation, while having a much thorough documentation available elsewhere. :-)